### PR TITLE
Documentation: Expand package manager FAQ with pkg info

### PR DESCRIPTION
**Pull request by kleinesfilmroellchen**
Explain a bit about what pkg is and what it isn't, and clarify that
while Serenity kinda-sorta has the beginnings of a "package manager",
it's not exactly like a typical Linux package manager with binary
software.

### CODE REVIEW

File: Documentation/FAQ.md [hunk 1]
```diff
@@ -28,11 +28,15 @@ The SerenityOS project tries to maximize hackability, accountability, and fun(!)
 
 ## Does SerenityOS have a package manager?
 
-No, SerenityOS does not have a package manager. The project uses a monorepo approach, meaning that all software is built in the same style and using the same tools. There is no reason to have something like a package manager because of this.
+To simplify: No, SerenityOS does not have a "Linux-style" package manager with pre-built software. The `pkg` tool interacts with the existing SerenityOS Ports system.
```
Comment (ADKaster) [inline Documentation/FAQ.md:31]: I don't like mentioning the pkg tool and the Port system before the paragraph with *However*. it seems confusing to reference a concept before introducing it.

I also don't think the "To simplify:" clause makes sense. Perhaps "In short, " ? or just omit

File: Documentation/FAQ.md [hunk 2]
```diff
@@ -28,11 +28,15 @@ The SerenityOS project tries to maximize hackability, accountability, and fun(!)
...
 
-*However* there are ports which can be found in the [Ports directory](../Ports). A port is a piece of software that can optionally be installed, might not have been built by us, but supports running on SerenityOS. They act quite similarly to packages, each coming with their own install script.
+More specifically: SerenityOS does not have a package manager that allows you to download pre-built software. The project uses a monorepo approach, meaning that all software is built in the same style and using the same tools. Additionally, the SerenityOS ABI, such as library symbols or syscall interfaces, have absolutely no stability guarantee and should be expected to change at any moment. There is no reason to have a binary package manager because of this.
```
Comment (ADKaster) [inline Documentation/FAQ.md:33]: replace colon with comma?

We should include a mention that "LibC is the ABI, similar to OpenBSD's guarantees" or some such. (assuming our guarantees are actually similar to OpenBSD, but I remember that being the assumption from discussions of porting the Go language). Though I see you mentioned that library symbols are not stable either, maybe that's safer to promise here.

File: Documentation/FAQ.md [hunk 3]
```diff
@@ -28,11 +28,15 @@ The SerenityOS project tries to maximize hackability, accountability, and fun(!)
...
 
-Currently when running the system in a virtual machine, ports need to be cross compiled on the host, and added to the file system image before booting. Then it's also possible to configure the build system to [in- or exclude components](./AdvancedBuildInstructions.md#component-configuration) from a build.
+*However* there are ports which can be found in the [Ports directory](../Ports). A port is a piece of software that can optionally be installed, might not have been built by us, but supports running on SerenityOS. They act quite similarly to packages, each coming with their own install script. The significant difference is that ports are always built from source.
```
Comment (ADKaster) [inline Documentation/FAQ.md:35]: Maybe now is a good time to reword this, now that there's a bit more context above.

"The supported method to use third-party software is by compiling ports that can be found in ...." ?

**Review by ADKaster — CHANGES_REQUESTED** [review]

File: Documentation/FAQ.md (hunk 2 above)
Comment (kleinesfilmroellchen) [reply Documentation/FAQ.md:33]: You're right, the LibC ABI thing should be mentioned; it also came up with Zig.

File: Documentation/FAQ.md (hunk 1 above)
Comment (kleinesfilmroellchen) [reply Documentation/FAQ.md:31]: I'll omit pgk here; whoever wants to read on will find out about pkg and the Ports soon enough.

**Review by ADKaster — APPROVED** [review]

**Pull request merged**